### PR TITLE
fix(PE-6064): Update signer readme

### PR DESCRIPTION
**Pull request by atticusofsparta**

### CODE REVIEW

**Comment by dtfiedler** [comment]
make this commit a `chore` so commitlint does not stage a new release

File: README.md [hunk 1]
```diff
@@ -196,7 +196,8 @@ Factory function to that creates a read-only or writeable client. By providing a
 // read-only client that has access to all read APIs
 const arIOReadable = ArIO.init()
 
-const browserSigner = new ArConnectSigner(window.arweaveWallet);
+const arweave = Arweave.init({ ...arweave api config })
```
Comment (dtfiedler) [inline README.md:199]: simplify this example

```suggestion
const arweave = Arweave.init({})
```

**Comment by atticusofsparta** [comment]
@dtfiedler like rollback and reapply the commit or just change the PR title?

File: src/common/contracts/warp-contract.ts [hunk 2]
```diff
@@ -68,7 +69,7 @@ export class WarpContract<T>
     logger?: Logger;
   }) {
     this.contractTxId = contractTxId;
-    this.contract = warp.contract(contractTxId);
+    this.contract = warp.use(new DeployPlugin()).contract(contractTxId);
```
Comment (dtfiedler) [inline src/common/contracts/warp-contract.ts:72]: not necessary yet

**Comment by dtfiedler** [comment]
> @dtfiedler like rollback and reapply the commit or just change the PR title?

yes - rec just creating a new branch off of main to avoid rebase/force push

File: README.md (hunk 1 above)
Comment (atticusofsparta) [reply README.md:199]: doing this has side affects with arweave-js - you cool with an ar-io.dev config in the example?

Comment (dtfiedler) [reply README.md:199]: what do you mean? where are arweave, api and config coming from in this example? not clear

Comment (atticusofsparta) [reply README.md:199]: it was a description of what to put - changing to this: 
```
const arweave = Arweave.init({
  host: 'ar-io.dev',
  port: 443,
  protocol: 'https'
})
```

**Pull request closed without merging**